inference-engine/src/vpu/graph_transformer/src/stages/roi_align.cpp

```diff
@@ -17,11 +17,18 @@ VPU_DECLARE_ENUM(ROIAlignMode,
     Max = 1
 )
 
+VPU_DECLARE_ENUM(ROIAlignStep,
+    Repacking = 0,
+    ROIAlignCHWc = 1,
+    ROIAlign = 2
+)
```

Comment:
`Step` in name of enum makes me think that all entries are subsequent stages of ROIAlign execution, but while there are 3 entries, I think ROIAlign is actually 2 step kernel (repack and then some kernel version). Assuming this, do you think it's more convenient to have different stage type for repacking?

Reply:
But as for name `Step`, ROIAlign can be 2-steps, or 1-step kernel. This repacking step is used for ROIAlign only. It is not an independent layer, so I decided to leave it as a part of ROIAlign layer.

Reply:
I think I see your point, but still there are 2 subsequent stages in model, right? So the fact that you place under one type does not change there is separate dependant stage (as for `LoopStart`/`LoopEnd`)
Also, I see that a lot of information from ROIAlign almost nothing is needed for repacking stage

Reply:
Repacking stage is a part of ROIAlignCHWc because it uses parameters of this layer to calculate tiling step `channel_step`, which is used for repacking. Making repacking as a separate stage will lead to code duplicating (calculating tiling steps) or make ROIAlign pipeline more complex.